### ISSUE TITLE
Open multiple window

### ISSUE THREAD

**Issue by folkevil** [duncan3dc/dusk#4]
Hi,
i would like to know how to open multiple windows ?

**Comment by duncan3dc** [comment]
Hi @folkevil,

I've not used this functionality myself, but it looks like you can just create multiple instances:

```php
use duncan3dc\Laravel\Dusk;

$client1 = new Dusk;
$client2 = new Dusk;

$client1->visit("https://www.google.co.uk/?q=one");
sleep(3);

$client2->visit("https://www.google.co.uk/?q=two");
sleep(3);

$client1->visit("https://www.google.co.uk/?q=three");
sleep(3);

$client2->visit("https://www.google.co.uk/?q=four");
sleep(3);
```

**Comment by folkevil** [comment]
@duncan3dc look like this solution didn't work because they used same port

``
 [Facebook\WebDriver\Exception\NoSuchDriverException]
  no such session
    (Driver info: chromedriver=2.32.498550 (9dec58e66c31bcc53a9ce3c7226f0c1c5810906a),platform=Windows NT 10.0.16299 x86_64)
``

**Comment by duncan3dc** [comment]
That must be a windows issue, let me see if I can make the port configurable for you

**Comment by duncan3dc** [comment]
I've requested a change [upstream](https://github.com/laravel/dusk/pull/430), if this is accepted then I'll be able to support it here, if they reject it I'll need to find another way to handle it

**Comment by duncan3dc** [comment]
Unfortunately upstream has rejected this feature (https://github.com/laravel/dusk/pull/449), but I'll take a look at adding support for it in this project

**Comment by duncan3dc** [comment]
This is now available in version [0.7.0](https://github.com/duncan3dc/dusk/releases/tag/0.7.0)
```php
use duncan3dc\Laravel\Drivers\Chrome;
use duncan3dc\Laravel\Dusk;

require __DIR__ . "/vendor/autoload.php";

$client1 = new Dusk(new Chrome(9516));
$client2 = new Dusk(new Chrome(9517));

$client1->visit("https://www.google.co.uk/?q=one");
sleep(3);

$client2->visit("https://www.google.co.uk/?q=two");
sleep(3);

$client1->visit("https://www.google.co.uk/?q=three");
sleep(3);

$client2->visit("https://www.google.co.uk/?q=four");
sleep(3);
```
